neo/Ledger/MemoryPool.cs

```diff
@@ -200,6 +200,21 @@ public IEnumerable<Transaction> GetSortedVerifiedTransactions()
             }
         }
 
+        /// <summary>
+        /// Clear mem pool
+        /// </summary>
+        public void Clear()
+        {
+            _txRwLock.EnterWriteLock();
+
+            _sortedTransactions.Clear();
```

Comment:
I think we should avoid clearing this collection. What do you think?

Reply:
If you want to clean all the pool, you need to clean all the collections